src/api/GitHubAPI.tsx

```diff
@@ -0,0 +1,110 @@
+import { Endpoints } from "@octokit/types";
+import { request } from "@octokit/request";
+import YAML from "yaml";
+import { ModelCardProps } from "./../components";
+
+export type ListIssueResponse =
+    Endpoints["GET /repos/{owner}/{repo}/issues"]["response"];
+export type IssueData =
+    Endpoints["GET /repos/{owner}/{repo}/issues/{issue_number}"]["response"]["data"];
+export type IssueLabelsData =
+    Endpoints["GET /repos/{owner}/{repo}/issues/{issue_number}"]["response"]["data"]["labels"];
+
+export async function GitHubIssues() {
+    const result = await request("GET /repos/{owner}/{repo}/issues", {
+        owner: "Layout-Parser",
+        repo: "platform",
+        state: "closed",

```

Comment:
I think searching based on the tag `model/approved` would be slightly better?

Reply:
but yeah, we can enforce it should be both closed and has the tag `model/approved`.